src/main/kotlin/no/hvl/dat250/app/dto/PollDTO.kt

```diff
@@ -34,6 +43,17 @@ fun PollRequest.toPoll(id: Long = -1): Poll {
   poll.secondAnswer = secondAnswer ?: ""
   return poll
 }
+fun PollCreateRequest.toPoll(id: Long = -1): Poll {
+  val poll = Poll()
+  poll.id = id
+  poll.startDate = startDateTime
+  poll.endDate = endDateTime
+  poll.private = private ?: false
+  poll.question = question ?: ""
+  poll.firstAnswer = firstAnswer ?: ""
+  poll.secondAnswer = secondAnswer ?: ""
```

Comment:
Disse trengs ikke, de vil aldri være `null`
```suggestion
  poll.question = question
  poll.firstAnswer = firstAnswer
  poll.secondAnswer = secondAnswer
```